### PR TITLE
Check for file existence before attempting to remove during cleanup

### LINKED ISSUE #1887: [BUG] cleanup command on Windows shows 'failures' for non-existent files.
cleanup does not properly handle non-existent files when performing this step on a host without a daemon install

```
INFO Removing CRC Machine Instance directory
Failed to kill tray process: exit status 1
remove C:\Users\gbraad\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\tray-windows.lnk: The system cannot find the file specified.
remove C:\Users\gbraad\.crc\bin\crc-daemon-autostart.bat: The system cannot find the file specified.
remove C:\Users\gbraad\.crc\bin\launch-crc-daemon.ps1: The system cannot find the file specified.
```

### PR DESCRIPTION
**Pull request by anjannath**
Fixes #1887

### CODE REVIEW

File: pkg/crc/preflight/preflight_checks_tray_windows.go [hunk 1]
```diff
@@ -100,8 +100,12 @@ func fixTrayInstalled() error {
 }
 
 func removeTray() error {
+	trayShortcutPath := filepath.Join(constants.StartupFolder, constants.TrayShortcutName)
 	_ = stopTray()
-	return goos.Remove(filepath.Join(constants.StartupFolder, constants.TrayShortcutName))
+	if os.FileExists(trayShortcutPath) {
```
Comment (gbraad) [inline pkg/crc/preflight/preflight_checks_tray_windows.go:105]: Please refactor to a single function that gets re-used:

```
RemoveFileIfExists(filepath string) err {
    if (filepath) {
       return goos.Remove(filepath)
    }
    return nil
}
```

**Review by gbraad — CHANGES_REQUESTED** [review]
unnecessary repetition...

File: pkg/crc/preflight/preflight_checks_tray_windows.go (hunk 1 above)
Comment (anjannath) [reply pkg/crc/preflight/preflight_checks_tray_windows.go]: Changed

**Review by cfergeau — APPROVED** [review]

**Review by gbraad — APPROVED** [review]
easy (as in clean) fix

**Pull request merged**